project.clj

```diff
@@ -16,10 +16,12 @@
   :dependencies [[org.clojure/clojure "1.7.0"]
 
                  ;; begin version conflict resolution dependencies
-                 [puppetlabs/typesafe-config "0.1.4"]
+                 [puppetlabs/typesafe-config "0.1.4" :exclusions [com.typesafe/config]]
                  [org.clojure/tools.macro "0.1.5"]
                  ;; end version conflict resolution dependencies
 
+                 [com.typesafe/config "1.3.0"]
```

Comment:
The version of com.typesafe/config we're using in our puppetlabs/typesafe-config wrapper is outdated by a couple minor versions. It doesn't preserve formatting or comments during file modifications.

Should be easy to upgrade our wrapper to the newer version and get rid of this override.